src/static/icons/Star.tsx

```diff
@@ -0,0 +1,37 @@
+import { neutral } from '@guardian/src-foundations/palette';
+import React from 'react';
+
+export const Star = ({
+	starId = '000',
```

Comment:
I _think_ we don't need this default?

Reply:
I think it would make it easier to debug, and allow consumers not to worry too much about what they’d use as an ID.